the-new-octal/src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>TheNewOctal</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="stylesheet" type="text/css"
+    href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:400,700|Material+Icons" />
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css">
+  <link rel="stylesheet" href="assets/css/material-dashboard.min.css">
```

Comment:
Majd megnézem a konzultációt ahol erre tettek javaslatot.
Az én tippem:
- vagy nem duplikáljuk le a template forrását 
- vagy csak a használt részeit
- vagy az angular számára library-ként hivatkozzuk be
Elkerülve az 57ezer soros kódmódosítást
![image](https://user-images.githubusercontent.com/10902799/153405976-e3adc9a2-a39e-4012-9449-3724c6a506f2.png)


![image](https://user-images.githubusercontent.com/10902799/153405850-95089194-f2af-4400-a726-b14f56bf3a7e.png)

Reply:
hiába raktam be az angular.json fájlba a hivatkozásokat a js fájlokra, nem működött;
most annyit tudtam tenni, hogy a html fájlban kikisérleteztem, melyik js fájlok kellenek a hamburger menünek, a többit töröltem a html-ből
az még nem derült ki, hogy pl a táblázatokhoz milyen fájlok kellenek, de ha készen van az egész, szerintem az assets-ből kitörölhetjük a felesleges fájlokat, mert ott pl vannak mintaoldalak, képek ilyesmi, ami majd nem kell